harness/propertyHelper.js

```diff
@@ -268,3 +269,32 @@ function verifyNotConfigurable(obj, name) {
     throw new Test262Error("Expected obj[" + String(name) + "] NOT to be configurable, but was.");
   }
 }
+
+/**
+ * Use this function to verify the properties of a primordial object.
+ * verifyPrimordialProperty accounts for the expected differences in the writable and configurable attributes when running under Hardened JavaScript's lockdown.
+ * For non-priordial objects, use verifyProperty.
```

Comment:
```suggestion
 * For non-primordial objects, use verifyProperty.
```

Reply:
Well... your recommended name change will give me more practice typing "primordial."

Reply:
Sorry for the unrelated comment, but I have to say as a keen observer of development in this repository... As a non-native speaker I have to google (merriam-websterize) these things 😄 This is very interesting direction especially for runtimes allowing interop access (Jint is a .NET interpreter that allows injecting .NET services and objects into runtime - there are sandboxing neends for big players wanting to support user-supplied scripting).